### PR TITLE
Fixing progress ring regression.

### PR DESCRIPTION
**Pull request by dhoehna**
## Summary of the pull request
My multi-account support regressed behavior when searching for repos.  This happened because the async command, ran an async action.  Because of the await, the UI threads wasn't updating.

Moved the code onto the UI thread.  Progress ring shows up.

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Review by krschau — APPROVED** [review]

File: tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs [hunk 1]
```diff
@@ -338,13 +338,16 @@ private void CancelButtonPressed()
     /// Switches the repos shown to the account selected.
     /// </summary>
     [RelayCommand]
-    private async Task MenuItemClick(string selectedItemName)
+    private void MenuItemClick(string selectedItemName)
```
Comment (AmelBawa-msft) [inline tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs:341]: Not very clear why this change fixes the UI. Async/await should help the UI be more responsive.

Comment (dhoehna) [reply tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs:341]: I believe it is because I'm calling `await MenuItemClick` from the ViewModel and somehow the code execution pauses at `GetRepositoriesAsync`  I do have a bug on me to move this to the code-behind thought since this is more UI logic, than ViewModel logic.

**Review by vineeththomasalex — APPROVED** [review]
